frontend-web/webclient/app/ui-components/Progress.tsx

```diff
@@ -151,10 +150,29 @@ export function NewAndImprovedProgress({
     style["--limit-bar-color-start"] = `var(--${limitColor.start})`;
     style["--limit-bar-color-end"] = `var(--${limitColor.end})`;
 
-    const warning = withWarning ? <OverallocationLink><TooltipV2 tooltip={UNABLE_TO_USE_FULL_ALLOC_MESSAGE}>
-        <Icon size={"20px"} mr="8px" name={"heroExclamationTriangle"} color={"warningMain"} />
-    </TooltipV2></OverallocationLink> : null;
-    return <Flex alignItems={"center"}>{warning}<div className={NewAndImprovedProgressStyle} data-label={label} style={style} /></Flex>
+    const warning = withWarning ? (
+        <div style={{position: "relative"}}>
+            <div style={{position: "absolute", left: "-20px"}}>
+                <OverallocationLink>
+                    <TooltipV2 tooltip={UNABLE_TO_USE_FULL_ALLOC_MESSAGE}>
+                        <Icon size="20px" mr="8px" name="heroExclamationTriangle" color="warningMain"/>
+                    </TooltipV2>
+                </OverallocationLink>
+            </div>
+        </div>
+    ) : null;
+
+    return (
+        <Flex alignItems="center">
+            <div className={NewAndImprovedProgressStyle} style={style}>
+                <div className="progress-limit-overlay"/>
+                {label && <span className="progress-label">
```

Comment:
I'd prefer if the classnames used were stored in variables (see https://github.com/SDU-eScience/UCloud/blob/master/frontend-web/webclient/app/Unstyled/index.ts#L183), but it's not important enough to block the PR.